TheMovieDatabase/Scenes/MovieDetail/MovieDetailViewController.swift

```diff
@@ -0,0 +1,254 @@
+
+        scrollView.addSubview(contentView)
+        contentView.edgesToSuperview()
+        contentView.widthToSuperview()
+        
+        contentView.addSubview(contentStackView)
+        contentStackView.edgesToSuperview(insets: .init(top: 0, left: 20, bottom: 0, right: 20))
+        
+        contentStackView.addArrangedSubview(ratingAndDateView)
+        ratingAndDateView.addSubview(ratingAndDateStackView)
+        ratingAndDateStackView.edgesToSuperview(excluding: .right)
+
+        ratingAndDateStackView.addArrangedSubview(imdbImageView)
+        imdbImageView.size(.init(width: 48, height: 48))
+        
+        ratingAndDateStackView.addArrangedSubview(starImageView)
+        starImageView.size(.init(width: 24, height: 24))
+        
+        ratingAndDateStackView.addArrangedSubview(movieRatingLabel)
+        ratingAndDateStackView.setCustomSpacing(0, after: movieRatingLabel)
+        
+        ratingAndDateStackView.addArrangedSubview(movieTopRatingLabel)
+        
+        ratingAndDateStackView.addArrangedSubview(yellowCircleImageView)
+        yellowCircleImageView.size(.init(width: 6, height: 6))
+    
+        ratingAndDateStackView.addArrangedSubview(movieDateLabel)
+        ratingAndDateStackView.setCustomSpacing(10, after: movieDateLabel)
+        
+        contentStackView.addArrangedSubview(movieInfoStackView)
+        movieInfoStackView.addArrangedSubview(movieTitleLabel)
+        movieInfoStackView.addArrangedSubview(movieOverviewLabel)
+        
+        view.addSubview(similarStackView)
+        similarStackView.topToBottom(of: scrollView).constant = 10
+        similarStackView.edgesToSuperview(excluding: .top, insets: .init(top: 0, left: 20, bottom: 0, right: 20), usingSafeArea: true)
+        similarStackView.addArrangedSubview(similarMoviesLabel)
+        similarStackView.addArrangedSubview(similarMoviesCollectionView)
+        similarMoviesCollectionView.height(160)
+    }
+}
+
+// MARK: - ConfigureContent
+extension MovieDetailViewController {
+    
+    private func configureContent() {
+        similarMoviesCollectionView.dataSource = self
+        similarMoviesCollectionView.delegate = self
+        similarMoviesCollectionView.register(SimilarMovieCell.self)
+    }
+    
+    private func setItem() {
+        movieImageView.setImage(Config.backdropPathBaseURL + (viewModel.backdropPath ?? ""))
+        movieRatingLabel.text = viewModel.movieRating
+        movieDateLabel.text = viewModel.date
+        movieTitleLabel.text = viewModel.title
+        movieOverviewLabel.text = viewModel.overview
+    }
+    
+    private func updateStarImage() {
+        guard let star = viewModel.starRating else { return }
+        if star >= 6.5 {
+            starImageView.image = .imgFilledStar
+        } else {
+            starImageView.image = .imgNotFilledStar
+        }
+    }
+    
+    private func updateSubviews() {
+        if viewModel.backdropPath == nil {
```

Comment:
boş olduğu durum için placeholder image tanımlanabilir yukarı almaya çok gerek yok gibi